tests/service/env_cleanup.yml

```diff
@@ -8,7 +8,9 @@
       - "HTTP/{{ nohost_fqdn }}"
       - HTTP/svc.ihavenodns.info
       - HTTP/no.idontexist.local
+      - HTTP/no.idontexist.info
```

Comment:
This change is unrelated.

Reply:
This change is in a TEMP commit, so service tests pass.